CHANGES.rst

```diff
@@ -8,6 +8,42 @@ https://rosettasciio.readthedocs.io/en/latest/changes.html
 
 .. towncrier release notes start
 
+0.8.0 (2025-03-28)
+==================
+
+Enhancements
+------------
+
+- Improve error message when loading spectrum image from :ref:`EMD Velox <emd_fei-format>` file and the ``sparse`` library is not installed. (`#305 <https://github.com/hyperspy/rosettasciio/issues/305>`_)
+- Add :func:`~.bruker.export_metadata` utility function for exporting metadata from :ref:`bruker-format` file. (`#326 <https://github.com/hyperspy/rosettasciio/issues/326>`_)
+- Add support for distributed lazy loading in :ref:`blockfile-format` reader and remove loading using threaded memory mapping which is discouraged in recent dask releases. (`#372 <https://github.com/hyperspy/rosettasciio/issues/372>`_)
+- Add support for distributed implementation to the :ref:`ripple <ripple-format>` reader. (`#376 <https://github.com/hyperspy/rosettasciio/issues/376>`_)
+
+
+Bug Fixes
+---------
+
+- Raise a warning instead of an error when the beam energy can't be found in :ref:`bruker-format` xrf files. (`#326 <https://github.com/hyperspy/rosettasciio/issues/326>`_)
```

Comment:
```suggestion
- Raise a warning instead of an error when the beam energy can't be found in :ref:`bruker-format` ``xrf`` files. (`#326 <https://github.com/hyperspy/rosettasciio/issues/326>`_)
```
Aligning with previous formatting of file endings in the release notes.